src/timidity/options.h

```diff
@@ -28,7 +28,8 @@
 #define DEFAULT_AMPLIFICATION 	70
 
 /* Default polyphony */
-#define DEFAULT_VOICES	32
+/* #define DEFAULT_VOICES	32 */
+#define DEFAULT_VOICES	256
```

Comment:
I don't know when / why SDL_mixer did this but isn't it overkill?

Reply:
Possibly. It seems to have been added to SDL_mixer in this commit: libsdl-org/SDL_mixer@2bcd3188c27cafe3a7f12a7548d58e4c233d2fc0

Reply:
@icculus: should we keep this as the old default or should we bump as SDL_mixer did?

History: `Timidity++` 1.0.0 to 1.3.10 has  DEFAULT_VOICES  32 and MAX_VOICES 256
v2.0 changes DEFAULT_VOICES to 64. v2.13.2 (from 2004) changes to DEFAULT_VOICES 
to 256, no MAX_VOICES.

Reply:
Also note: what we have here is practically `timidity-0.2i` -- `Timidity++` has more features packed in.